app/models/service_retire_task.rb

```diff
@@ -68,6 +70,10 @@ def create_task(svc_rsc, parent_service, nh, parent_task)
 
   private
 
+  def retire_subtasks?(parent_svc)
+    parent_svc.kind_of?(ServiceAnsiblePlaybook) && !parent_svc.retireable? ? false : true
```

Comment:
I think this method should call `retain_resources_on_retirement?` based on comments about `retireable?`.  It will need to handle if the parent_service exposes this method.